core/server/data/migrations/versions/2.3/1-add-webhook-columns.js

```diff
@@ -0,0 +1,38 @@
+const common = require('../../../../lib/common');
+const commands = require('../../../schema').commands;
+const table = 'webhooks';
+const newColumnNames = ['name', 'integration_id', 'secret', 'last_triggered_at', 'api_version'];
+
+function printResult(operation, columnName) {
+    return `${operation} column ${columnName} in ${table} table`;
+}
+
+module.exports.up = (options) => {
+    const connection = options.connection;
+    return Promise.map(newColumnNames, (newColumnName) => {
+        return connection.schema.hasColumn(table, newColumnName)
+            .then((exists) => {
+                if (exists) {
+                    common.logging.warn(printResult('Adding', newColumnName));
+                    return;
+                }
+                common.logging.info(printResult('Adding', newColumnName));
+                return commands.addColumn(table, newColumnName, connection);
+            });
+    }, {concurrency: 100});
```

Comment:
concurrency 100 is a little weird here, because we only add 5 columns 😁

Reply:
```suggestion
    });
```